### PR TITLE
[POSUI-13] Add Broadcast to reflect CVV enter

### PR DESCRIPTION
**Pull request by DhruboPaulPax**

### CODE REVIEW

File: lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/status/PINStatus.java [hunk 1]
```diff
@@ -27,13 +27,11 @@ private PINStatus(){
 
     /**
      * Broadcast Action: PIN_ENTER_COMPLETED<br>{@value #PIN_ENTER_COMPLETED}<br>
```
Comment (YaninaYang) [inline lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/status/PINStatus.java:29]: add description for this event

File: lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/status/PINStatus.java [hunk 2]
```diff
@@ -27,13 +27,11 @@ private PINStatus(){
...
-     * @deprecated Never used
      */
     public static final String PIN_ENTER_COMPLETED = "com.pax.us.pay.PIN_ENTER_COMPLETED";
 
     /**
      * Broadcast Action: PIN_ENTER_ABORTED<br>{@value #PIN_ENTER_ABORTED}<br>
```
Comment (YaninaYang) [inline lib-ui-constant/src/main/java/com/pax/us/pay/ui/constant/status/PINStatus.java:34]: add description for this event

**Review by YaninaYang — CHANGES_REQUESTED** [review]

**Review by YaninaYang — APPROVED** [review]
approved on 09/26/2023

**Review by Jolie-Yanglj — APPROVED** [review]
Approved 09/26/2023

**Pull request merged**